docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+  web:
+    build:
+      context: .
+    image: nuxt_dev
+    container_name: nuxt_dev
```

Comment:
Can we have a different name here instead of `nuxt_dev`.
Maybe something like "api_party"??

I am not sure about the limitations of name in image and container_name, but I would love to see something other than nuxt_dev